dotnet/src/Skills/Skills.OpenAPI/Extensions/KernelAIPluginExtensions.cs

```diff
@@ -101,7 +101,8 @@ public static class KernelAIPluginExtensions
             skillName,
             httpClient,
             executionParameters,
-            cancellationToken).ConfigureAwait(false);
+            uri,
+            cancellationToken: cancellationToken).ConfigureAwait(false);
```

Comment:
avoid extra parameters verbosity.
```suggestion
            cancellationToken).ConfigureAwait(false);
```